docs/user-guide/writing-your-docs.md

```diff
@@ -179,25 +213,27 @@ Left         | Center        | Right
 Left         | Center        | Right
 ```
 
-#### Fenced code blocks
+### Fenced code blocks
 
-Start with a line containing 3 or more backtick \` characters, and ends with the first line with the same number of backticks \`:
+The first line should contain 3 or more backtick (`` ` ``) characters, and the
+last line should contain the same number of backtick characters (`` ` ``):
```

Comment:
I reworked this paragraph. It made no sense in the previous form. Not specifically a linter issue. Although, the backticks being in code spans rather than escaped seems better to me.